spec/system/comments/user_edits_a_comment_spec.rb

```diff
@@ -4,10 +4,15 @@
   let(:user) { create(:user) }
   let!(:article) { create(:article, show_comments: true) }
   let(:new_comment_text) { Faker::Lorem.paragraph }
-  let!(:comment) { create(:comment, commentable: article, user: user, body_markdown: Faker::Lorem.paragraph) }
+  let!(:comment) do
+    create(:comment,
+           :with_path,
+           commentable: article,
+           user: user,
+           body_markdown: Faker::Lorem.paragraph)
+  end
 
   before do
-    Notification.send_new_comment_notifications(comment)
```

Comment:
Not sure why this was here, the specs pass fine without it so I removed it, figured it would save us some time.